mention requestor in GET of acecsspackageassignmentrequest (#13855)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accesspackageassignmentrequest-get.md

## Optional query parameters

> This method supports some of the OData query parameters to help customize the response. For example, to retrieve the access package that was requested, include `$expand=accessPackage` in the query. To retrieve the requestor as well, include `$expand=requestor` in the query.  To retrieve the resulting assignment, include `$expand=accessPackageAssignment` in the query.  For general information, see [OData query parameters](/graph/query-parameters).

Reviewer: Proposing that you add the **Supports $expand** statement to the description of the relationships. See [this commit](https://github.com/microsoftgraph/microsoft-graph-docs/pull/13868/commits/2897550ca420714725a7991390b62768daefa296). Allows you make 1 reference for all + use this space for more complex syntaxes.
```suggestion
This method supports the `$expand` OData query parameter to expand the relationships. For example, to retrieve the access package that was requested, include `$expand=accessPackage` in the query. For general information, see [OData query parameters](/graph/query-parameters).
```
